cmd/cclient/main.go

```diff
@@ -86,10 +89,28 @@ func send(recipient string, amountStr string) {
 	log.Printf("transaction %d cleared", transaction.Sequence)
 }
 
+func handler(w http.ResponseWriter, r *http.Request) {
+	pass :=strings.TrimLeft(r.URL.Path, "/")
+	kp := util.NewKeyPairFromSecretPhrase(pass)
+	s := status(kp.PublicKey())
+	if s != nil {
+		fmt.Fprintf(w, "{ \"sequence\": %v, \"balance\": %v }",
```

Comment:
i think you want the %v here to be more specific, like %d

Reply:
Will do.